src/styles/_global-vars.scss

```diff
@@ -0,0 +1,2 @@
+$primaryColor: hsl(var(--primary-color-h), var(--primary-color-s), var(--primary-color-l));
+$accentColor: hsl(var(--accent-color-h), var(--accent-color-s), var(--accent-color-l));
```

Comment:
So we have two places that store these vars: common libs and this repo?

Reply:
This is a standalone web component that's going to be used in a variety of apps (OpenPaaS, LinShare, and Twake), so it might not make sense to require the color vars from OpenPaaS because it would increase the amount of coupling. They might have similar names (actually I shouldn't have done that), but they should not really be the same thing.

Reply:
Ok